cmd/cli/cmd/alpha/content/terraform.go

```diff
@@ -0,0 +1,52 @@
+package content
```

Comment:
The interactive mode would be useful - but IMHO it can be implemented in a follow-up.